superset/assets/spec/javascripts/dashboard/Controls_spec.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { describe, it, beforeEach } from 'mocha';
+import { expect } from 'chai';
+
+import Controls from '../../../javascripts/dashboard/components/Controls';
+import { dashboardData } from './fixtures';
+
+describe('Controls', () => {
+  let wrapper;
+  const mockedProps = {
+    dashboard: dashboardData,
+  };
+  mockedProps.dashboard.css = 'foo';
+
+  beforeEach(() => {
+    wrapper = mount(<Controls {...mockedProps} />);
+  });
+
+  it('is a valid element', () => {
+    expect(React.isValidElement(<Controls {...mockedProps} />)).to.equal(true);
+  });
+
+  it('sets the state to cssTemplates on componentWillMount', () => {
```

Comment:
for this method we want to check that once the xhr request returns that it updates the css key on the state. here's an example of testing a method with an xhr request. https://github.com/airbnb/superset/blob/master/superset/assets/spec/javascripts/explore/actions_spec.js#L27-L28